modules/gui/bonus.lua

```diff
@@ -146,6 +181,8 @@ Gui.element{
         end
     end
 
+    disp[disp['bonus_display_personal_battery_recharge'].tags.counter].caption = format_number(disp['bonus_display_personal_battery_recharge'].slider_value)
+
```

Comment:
The code you currently have causes and error, and when fixed it doesn't reset the value. So you need the code below.
```suggestion
    local slider = disp['bonus_display_personal_battery_recharge_slider']
    slider.slider_value = config.player_special_bonus['personal_battery_recharge'].value
    disp[slider.tags.counter].caption = format_number(slider.slider_value)

```